lib/ProductOpener/Products.pm

```diff
@@ -173,9 +173,9 @@ sub init_product($) {
 					push @{$product_ref->{$field . "_tags" }}, get_taxonomyid($tag);
 				}
 			}
-			# if lc is not defined or is set en, set lc to main language of country
-			if ($lc eq 'en') {
-				$lc = $country_languages{lc($country)}[0];
+			# if lc is not defined or is set to en, set lc to main language of country
```

Comment:
Unrelated to #1145 though? Committed to wrong branch?

Reply:
Yes, same, it's bug https://github.com/openfoodfacts/openfoodfacts-server/issues/1177 which needed to be fixed. I was too lazy to make separate branches.